lib/ui/views/login_view.dart

```diff
@@ -73,18 +88,21 @@ class _LoginViewState extends State<LoginView> {
                                           width: borderRadiusOnFocus)),
                                   focusedErrorBorder: OutlineInputBorder(
                                       borderSide: BorderSide(
-                                          color: Colors.amberAccent,
+                                          color: getErrorTextColor(
+                                              model.getAppTheme()),
```

Comment:
```suggestion
                                          color: errorTextColor,
```

If you apply the change below 👇